index.js

```diff
@@ -64,6 +64,7 @@ const runAction = () => {
 		log("Logging in to Snapcraft…");
 		writeFileSync(LOGIN_FILE_PATH, process.env.INPUT_SNAPCRAFT_TOKEN);
 		run(`snapcraft login --with ${LOGIN_FILE_PATH}`);
+		run(`rm ${LOGIN_FILE_PATH}`);
```

Comment:
I'd prefer using the Node.js function instead: [`unlinkSync(LOGIN_FILE_PATH)`](https://nodejs.org/api/fs.html#fs_fs_unlinksync_path)